components/feature/top-sites/src/main/java/mozilla/components/feature/top/sites/DefaultTopSitesStorage.kt

```diff
@@ -67,7 +68,8 @@ class DefaultTopSitesStorage(
 
     override suspend fun getTopSites(
         totalSites: Int,
-        includeFrecent: Boolean
+        includeFrecent: Boolean,
+        frecencyThreshold: FrecencyThresholdOption
```

Comment:
Does this option change? Maybe just put it in the constructor or leave it internal as part of our implicit feature expectations.

Reply:
It doesn't change, but we also fetch the option from the TopSiteConfig